.github/workflows/release_nightly.yml

```diff
@@ -220,7 +220,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ needs.create-nightly-release.outputs.upload_url }}
-          asset_path: ./web/packages/extension/dist/firefox_unsigned.xpi
+          asset_path: ./web/packages/extension/dist/firefox.xpi
```

Comment:
I see mentions to `firefox_unsigned.xpi` in [`package.json`](https://github.com/ruffle-rs/ruffle/blob/master/web/packages/extension/package.json#L10). Should they be changed to `firefox.xpi`?

Reply:
The first steps still create `firefox_unsigned.xpi`:
`node tools/zip.js dist/firefox_unsigned.xpi`

`firefox_unsigned.xpi` is uploaded to the Mozilla server where it is signed, and then the signed copy is downloaded into `firefox.xpi`:
node tools/sign_xpi.js dist/firefox_unsigned.xpi dist/firefox.xpi

Reply:
So don't you need to provide the `MOZILLA_API_KEY`, `MOZILLA_API_SECRET` and `FIREFOX_EXTENSION_ID` environment variables, so that `sign_xpi.js` actually signs the extension?

Reply:
Yep, I'll set those as secrets in the GitHub settings for the Ruffle org. They'll be passed along to the CI environment.